### ISSUE TITLE
Missing translations in CrowndIn

### ISSUE THREAD

**Issue by tygryk** [moneymanagerex/moneymanagerex#3797]
MMEX version:
 - [x] 1.5.x
 - [x] 1.6.x

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [x] Mac OSX
 - [x] Linux 

# Description of the bug

Everything you see in English is missing from translation at CrowdIn. Or maybe I looked at wrong places.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

A picture is worth a thousand words

<img width="466" alt="Screen Shot 2021-08-20 at 21 51 42" src="https://user-images.githubusercontent.com/9718911/130281011-faa99e87-a2b6-4d9e-ada7-4b6dfcf8726b.png">
<img width="504" alt="Screen Shot 2021-08-20 at 21 52 38" src="https://user-images.githubusercontent.com/9718911/130281019-78a2514f-73c1-4b81-865d-cf15c2e74be4.png">
<img width="506" alt="Screen Shot 2021-08-20 at 21 53 17" src="https://user-images.githubusercontent.com/9718911/130281027-441a68c6-f604-4dde-9c14-58fa7ae934a3.png">
<img width="876" alt="Screen Shot 2021-08-20 at 21 53 49" src="https://user-images.githubusercontent.com/9718911/130281028-32a21ad1-c24e-4189-8d73-103ff9a91bd3.png">

Expected result:

Strings are there and translatable.

Actual result:

They are not.

N/A

**Comment by whalley** [comment]
Related: https://github.com/moneymanagerex/moneymanagerex/issues/2912

**Comment by renato-mmex** [comment]
I would say this is a low priority, better said it is not useful from my point of view.
I hardly think that a user who will do these important points that are mentioned here, in the sense of MMEX. 
It is common in the industry that licenses and privacy are created in the language of the program manufacturer (mostly in English).

**Comment by tygryk** [comment]
np, let it be so

However I would suggest having the "transaction" word translated (last screenshot). So it looks more consistent

**Comment by whalley** [comment]
> However I would suggest having the "transaction" word translated (last screenshot). So it looks more consistent

Yep will get that fixed.

**Comment by whalley** [comment]
@Vomikan - This will also have the effect of changing the attachment path location. I remember looking at this and thinking we should perhaps leave the existing directory name as-is but add a third element for the translated name. Otherwise as the translations are added the path to existing attachments will be lost. (possibly the reason why "Recurring Transaction" is "RecurringTransaction", i.e. to avoid space in directory name)

`{Model_Attachment::TRANSACTION, "Transaction", wxTRANSLATE("Transaction") },`

**Comment by tygryk** [comment]
@whalley interesting: what will happen after mmex restart if selected language is changed?

**Comment by whalley** [comment]
> @whalley interesting: what will happen after mmex restart if selected language is changed?

Nothing good I suspect!

**Comment by vomikan** [comment]
wxTRANSLATE("Transaction")  ==  "Transaction"

**Comment by whalley** [comment]
Good point. I’m mixing up this and _(“Translate”)
